### DISCUSSION TITLE
Schema isn't loaded in vscode

### DISCUSSION THREAD

**Discussion by rajzik** [renovatebot/renovate#12056] [Request Help]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

Azure DevOps (dev.azure.com)

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Latest

### Describe the bug

In visual studio code there is an error loading schema.

![image](https://user-images.githubusercontent.com/10364836/136154337-58dfbd01-bbc3-4709-b39a-fcb7c915edb3.png)

Seems like this can be related: https://github.com/microsoft/tsdoc/issues/253

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by HonkingGoose** [comment]
I cannot reproduce this on a fresh Gitpod container, with VSCode version `1.60.2`

The link to the schema file is valid as well, as far as I can tell: https://docs.renovatebot.com/renovate-schema.json

**Reply by rarkins** [reply]
Converted this to a discussion
